Bottom: b7cadaddaa3d09b620807b42f183be5e2dcc9269
Top:    9cca0d7b8efd4351af12572edb06d08fab4dfc3a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-27 02:41:09 +0100

Refresh of add-soldermask-layers

---

diff --git a/src/const.h b/src/const.h
index e6e850c..5ee3bc6 100644
--- a/src/const.h
+++ b/src/const.h
@@ -42,7 +42,7 @@
 #define TOP_SIDE                1
 
 
-#define EXTRA_LAYERS 2 /* 2x silkscreen layers */
+#define EXTRA_LAYERS 4 /* 2x silkscreen, 2x soldermask layers */
 
 /* ---------------------------------------------------------------------------
  * the layer-numbers of the two additional special (silkscreen) layers
